generators/profilegen/src/resources/templates/rollup/client_template.template

```diff
@@ -2,45 +2,23 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-<%- if @individual_gem_profile -%>
-require 'profiles/<%= @profile_name.downcase %>/<%= @module_definition_file_name.gsub('.rb','') %>'
-<%- @file_names.each do |file_name| -%>
-require 'profiles/<%= @profile_name.downcase %>/modules/<%= file_name %>'
-<%- end -%>
-require 'profiles/common/configurable'
-require 'profiles/common/default'
-<%- else -%>
 require '<%= @profile_name.downcase %>/<%= @module_definition_file_name.gsub('.rb','') %>'
 <%- @file_names.each do |file_name| -%>
 require '<%= @profile_name.downcase %>/modules/<%= file_name %>'
```

Comment:
similar comment to the one above here, it'd be great to simplify the template a bit, so it's more readable - anything that could help with that would be appreciated for future maintenance.